src/applications/appeals/10182/components/ContactInformation.jsx

```diff
@@ -107,6 +107,7 @@ export const ContactInfoDescription = ({ formContext, profile }) => {
                 title="Mobile phone number"
                 fieldName={FIELD_NAMES.MOBILE_PHONE}
                 deleteDisabled
+                noAlertClosing
```

Comment:
I'm not crazy about this name and think that something like `alertClosingDisabled` since we are using `deleteDisabled`

Reply:
Sounds good to me... naming is always hard :D